drivers/hmis/app/graphql/types/forms/enums/pick_list_type.rb

```diff
@@ -35,5 +35,7 @@ class Forms::Enums::PickListType < Types::BaseEnum
     value 'ENROLLMENTS_FOR_CLIENT', 'Enrollments for the client, including WIP and Exited.'
     value 'OPEN_HOH_ENROLLMENTS_FOR_PROJECT', 'Open HoH enrollments at the project.'
     value 'CE_EVENTS', 'Grouped HUD CE Event types'
+    value 'AUDIT_EVENT_RECORD_TYPES'
```

Comment:
I think this should be called `ENROLLMENT_AUDIT_EVENT_RECORD_TYPES` since it's specifically records that are related to enrollments